### PR TITLE
Mcp 1913 - Fix Spotless Check warnings for service-data-access main code.

### PR DESCRIPTION
**Pull request by wlin2019**
Fix Spotless Check warnings for service-data-access main code.

## Description

Spotless check reported 58 warnings in service-data-access main code. This PR fixes most of them. However, spotless check seems not to be updated for some new Java features, e.g., switch/yield. 

### Jira Tickets

- [MCP-000](https://amida.atlassian.net/browse/MCP-000)

## How to test this PR

- Step 1
- Step 2 

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
- [ ] If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) and
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
- [ ] If changing software behavior, post a link to the PR in Slack channel #benefits-virtual-regional-office

### CODE REVIEW

**Review by au2 — APPROVED** [review]

File: service-data-access/src/main/java/gov/va/vro/abddataaccess/model/LighthouseTokenMessage.java [hunk 1]
```diff
@@ -12,11 +14,12 @@
 @Setter
 @Getter
 @NoArgsConstructor
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
 public class LighthouseTokenMessage {
-  private String access_token;
-  private String token_type;
+  private String accessToken;
```
Comment (wlin2019) [inline service-data-access/src/main/java/gov/va/vro/abddataaccess/model/LighthouseTokenMessage.java:19]: Will it fail the test?

File: service-data-access/src/test/java/gov/va/vro/abddataaccess/exception/AbdExceptionTest.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+class AbdExceptionTest {
+
+  @Test
+  public void test() {
+    final AbdException ex1 = new AbdException();
+    final AbdException ex2 = new AbdException("test");
+    final AbdException ex3 = new AbdException(ex2);
+    final AbdException ex4 = new AbdException("test", ex1);
+
+    assertTrue(!ex1.getMessage().isEmpty());
```
Comment (dimitri-amida) [inline service-data-access/src/test/java/gov/va/vro/abddataaccess/exception/AbdExceptionTest.java:18]: How about assertFalse ?

**Review by dimitri-amida — COMMENTED** [review]
Looks good but the build is failing. Are these all the spotless checks or a subset?

File: service-data-access/src/test/java/gov/va/vro/abddataaccess/exception/AbdExceptionTest.java (hunk 2 above)
Comment (wlin2019) [reply service-data-access/src/test/java/gov/va/vro/abddataaccess/exception/AbdExceptionTest.java]: done.

**Review by dimitri-amida — APPROVED** [review]
it worked!

**Pull request merged**